src/client/components/forms/userCollection.js

```diff
@@ -262,7 +262,8 @@ class UserCollectionForm extends React.Component {
 							<div className={errorAlertClass}>
 								<Alert bsStyle="danger">Error: {errorText}</Alert>
 							</div>
-							<div className="text-center">
+							<div className="row">
+								<div className = "col-sm-6">
```

Comment:
You can add a simple Bootstrap class to add a space on top of both of these `col-sm-6` elements, which will give a bit of space between the buttons when they are on top of each other:

```suggestion
								<div className = "col-sm-6 margin-top-d5">
```

With that simply change on both elements, you can remove the changes to the `style.less` file.